recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - x86_64-microarch-level {{ microarch_level }}  # [unix and x86_64]
+    - x86_64-microarch-level {{ microarch_level }}  # [unix and x86_64 and (build_platform == target_platform)]
```

Comment:
Should hopefully address https://github.com/conda-forge/microarch-level-feedstock/issues/11.

Reply:
It did not.

Reply:
c.f. https://github.com/conda-forge/conda-forge.github.io/pull/2197